sdk/dotnet/CertificateAuthority/CaPool.cs

```diff
@@ -43,6 +43,132 @@ namespace Pulumi.Gcp.CertificateAuthority
     /// 
     /// }
     /// ```
+    /// ### Privateca Quickstart
```

Comment:
Sounds like you only updated the docker image in the tests? Are these SDK changes expected?

Reply:
I did regenerate the SDKs because I was debugging the tests so this is a bit odd - let me remove these